sql_test.py

```diff
@@ -0,0 +1,40 @@
+import mysql.connector
+from mysql.connector import errorcode
+
+# Obtain connection string information from the portal
+config = {
+    'host':'mictestdb.mysql.database.azure.com',
+    'user':'c_hawk7397',
+    'password':'m1(|<3yM0i_i53',
+    'database':'mictestdb',
+    'client_flags': [mysql.connector.ClientFlag.SSL],
+    'ssl_ca': 'DigiCertGlobalRootCA.crt.pem'
```

Comment:
Similar to the password concerns, we will also want this somewhere safe and accessible. Again it seems like it is an issue to solve when we get to hosting via the MIC team.